datamodel/referencedata.d/unitofmeasures.csv

```diff
@@ -1,9 +1,9 @@
-Unit of Measure Code,Unit of Measure Description
-KGM,Kilogram
-LBR,Pounds
-CBM,Cubic Metres
-CFT,Cubic Feet
-CEL,Celsius
-FAH,Fahrenheit
-FOT,Foot
-MTR,Meter
+Unit of Measure Code,Unit of Measure Description
+KGM,Kilogram
+LBR,Pounds
+CBM,Cubic Metres
+CFT,Cubic Feet

```

Comment:
These are the wrong values?!?

Reply:
And I am not quite sure what has changed in this file - it appers everything has been updated...?

Reply:
Updated. Don't know why this was reverted.